armi/materials/tests/test_uZr.py

```diff
@@ -13,16 +13,117 @@
 # limitations under the License.
 
 """Tests for simplified UZr material."""
-import unittest
+from unittest import TestCase
 
-from armi.materials.tests import test_materials
 from armi.materials.uZr import UZr
 
 
-class UZR_TestCase(test_materials._Material_Test, unittest.TestCase):
+class UZR_TestCase(TestCase):
+
     MAT_CLASS = UZr
 
+    def setUp(self):
+        self.mat = self.MAT_CLASS()
+
+    def test_isPicklable(self):
+        """Test that materials are picklable so we can do MPI communication of state.
+
+        .. test:: Test the material base class.
+            :id: T_ARMI_MAT_PROPERTIES0
+            :tests: R_ARMI_MAT_PROPERTIES
```

Comment:
This doesn't feel relevant to the linked requirement. Let's chat offline.

Reply:
I updated the comment to show that it does.